assets/scripts/segments/resizing.js

```diff
@@ -68,6 +68,11 @@ export function resizeSegment (el, resizeType, width, updateEdit, palette, initi
     width = normalizeSegmentWidth(width, resizeType)
   }
 
+  document.body.classList.add('immediate-segment-resize')
+  window.setTimeout(function () {
+    document.body.classList.remove('immediate-segment-resize')
+  }, SHORT_DELAY)
+
```

Comment:
Since this is repeated in a few spots, would it be better to make a small reusable function for it?